packages/kbn-search-connectors/types/connectors_api.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// TODO: delete this once ES client can be used for Connectors API
+
+enum Result {
```

Comment:
Enums in JavaScript are integer-based by default, so this enum right now has the values:
```
enum Result {
  created: 0,
  updated: 1,
etc etc
}
```

You'll need to explicity define all of these, assuming you want the actual string values here :)

Reply:
Alternatively you could do this with a string union ( ‘updated’ | ‘created’ | etc)